src/metabase/util/malli/fn.clj

```diff
@@ -295,8 +295,40 @@
   `(let [~'&f ~(deparameterized-fn-form parsed)]
      (core/fn ~@(instrumented-fn-tail error-context (fn-schema parsed)))))
 
+(defn uninstrumented-fn-form
+  "Given a `fn-tail` like
+
+  ([x :- :int y] (+ 1 2))
+
+  and parsed by [[parsed-fn-tail]],
+
+  return an unevaluated, and uninstrumented fn form like
+
+  (fn [x y] (+ 1 2))
+
+  This is used to emit fn which ignore checks."
+  [parsed]
+  (deparameterized-fn-form parsed))
+
+;; ------------------------------ Skipping Namespace Enforcement in prod ------------------------------
+
+;; FIXME: is there a cleaner way to setup `namespaces-toskip`?
+;; I was thinking we could `swap!` it from the namespaces themselves with:
+;; `(swap namespaces-toskip conj *ns*)`
+(require ['metabase.lib.metadata.calculation])
+
+(def namespaces-toskip
+  "Used to track namespaces to not enforce malli schemas on with `mu.fn/fn`."
+  (atom #{(find-ns 'metabase.lib.metadata.calculation)}))
```

Comment:
Open to suggestions about a clean way to do this, as well as which namespaces we should toskip.

Reply:
Maybe a metadata on namespace directly? Like `^:malli/skip`, for example?

Reply:
Yeah, I think decorating the namespaces makes sense. On the other hand we probably want this for all of `metabase.lib.*` and that's a lot of files to remember to do that in. Though I guess we could use a linter to require either `^:malli/skip` or `^:malli/enforce` within `metabase.lib.*`.

Reply:
I don't think I would operate on namespaces as objects, just on the symbol or string of the name. Have the set of namespaces be symbols or regexes rather than `clojure.lang.Namespace` objects.